### ISSUE TITLE
bug: builder tests failing 

### ISSUE THREAD

**Issue by andorsk** [TBD54566975/ssi-sdk#132]
**Expected**: Tests are passing
**Actual**: Builder tests are failing
**Environments**: CI/CD (github) and Mac OSX
**Steps to reproduce:**
1. CI/CD will fail if you submit it there OR
2. mage test will ail locally. 
**Logs:**
```ok  	github.com/TBD54566975/ssi-sdk/credential/exchange	0.378s
time="2022-07-04T23:27:06Z" level=error msg="credential manifest not valid against schema" error="Could not read schema from HTTP, response status is 404 Not Found"
time="2022-07-04T23:27:06Z" level=error msg="credential manifest not ready to be built" error="manifest failed json schema validation: Could not read schema from HTTP, response status is 404 Not Found"
time="2022-07-04T23:27:06Z" level=error msg="credential manifest not valid against schema" error="Could not read schema from HTTP, response status is 404 Not Found"
time="2022-07-04T23:27:06Z" level=error msg="credential manifest not ready to be built" error="manifest failed json schema validation: Could not read schema from HTTP, response status is 404 Not Found"
--- FAIL: TestCredentialManifestBuilder (1.30s)
    builder_test.go:[10](https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true#step:6:11)2: 
        	Error Trace:	/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder_test.go:102
        	Error:      	Received unexpected error:
        	            	Could not read schema from HTTP, response status is 404 Not Found
        	            	manifest failed json schema validation
        	            	github.com/TBD54566975/ssi-sdk/credential/manifest.(*CredentialManifest).IsValid
        	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/model.go:36
        	            	github.com/TBD54566975/ssi-sdk/credential/manifest.(*CredentialManifestBuilder).Build
        	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder.go:34
        	            	github.com/TBD54566975/ssi-sdk/credential/manifest.TestCredentialManifestBuilder
        	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder_test.go:101
        	            	testing.tRunner
        	            		/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1259
        	            	runtime.goexit
        	            		/opt/hostedtoolcache/go/1.17.11/x64/src/runtime/asm_amd64.s:1581
        	            	credential manifest not ready to be built
        	            	github.com/TBD54566975/ssi-sdk/util.LoggingErrorMsg
        	            		/home/runner/work/ssi-sdk/ssi-sdk/util/errors.go:24
        	            	github.com/TBD54566975/ssi-sdk/credential/manifest.(*CredentialManifestBuilder).Build
        	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder.go:35
        	            	github.com/TBD54566975/ssi-sdk/credential/manifest.TestCredentialManifestBuilder
        	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder_test.go:101
        	            	testing.tRunner
        	            		/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1259
        	            	runtime.goexit
        	            		/opt/hostedtoolcache/go/1.17.11/x64/src/runtime/asm_amd64.s:1581
        	Test:       	TestCredentialManifestBuilder
    builder_test.go:103: 
        	Error Trace:	/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/builder_test.go:103
        	Error:      	Should NOT be empty, but was <nil>
        	Test:       	TestCredentialManifestBuilder
time="2022-07-04T23:27:06Z" level=error msg="credential application not valid against schema" error="credential_application.format: Invalid type. Expected: object, given: null"
time="2022-07-04T23:27:06Z" level=error msg="credential application not ready to be built" error="application failed json schema validation: credential_application.format: Invalid type. Expected: object, given: null"
time="2022-07-04T23:27:06Z" level=error msg="credential fulfillment not valid against schema" error="credential_fulfillment.descriptor_map: Invalid type. Expected: array, given: null"
time="2022-07-04T23:27:06Z" level=error msg="credential fulfillment not ready to be built" error="fulfillment failed json schema validation: credential_fulfillment.descriptor_map: Invalid type. Expected: array, given: null"
time="2022-07-04T23:27:06Z" level=error msg="credential manifest not valid against schema" error="Could not read schema from HTTP, response status is 404 Not Found"
--- FAIL: TestCredentialManifest (0.04s)
    --- FAIL: TestCredentialManifest/Credential_Manifest_Vector_1 (0.03s)
        model_test.go:37: 
            	Error Trace:	/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/model_test.go:37
            	Error:      	Received unexpected error:
            	            	Could not read schema from HTTP, response status is 404 Not Found
            	            	manifest failed json schema validation
            	            	github.com/TBD54566975/ssi-sdk/credential/manifest.(*CredentialManifest).IsValid
            	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/model.go:36
            	            	github.com/TBD54566975/ssi-sdk/credential/manifest.TestCredentialManifest.func1
            	            		/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/model_test.go:37
            	            	testing.tRunner
            	            		/opt/hostedtoolcache/go/1.17.[11](https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true#step:6:12)/x64/src/testing/testing.go:[12](https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true#step:6:13)59
            	            	runtime.goexit
            	            		/opt/hostedtoolcache/go/1.17.11/x64/src/runtime/asm_amd64.s:1581
            	Test:       	TestCredentialManifest/Credential_Manifest_Vector_1
--- FAIL: TestCredentialManifestSchemas (0.26s)
    schema_test.go:17: 
        	Error Trace:	/home/runner/work/ssi-sdk/ssi-sdk/credential/manifest/schema_test.go:17
        	Error:      	Received unexpected error:
        	            	Could not read schema from HTTP, response status is 404 Not Found
        	Test:       	TestCredentialManifestSchemas
FAIL
FAIL	github.com/TBD54566975/ssi-sdk/credential/manifest	1.675s
ok  	github.com/TBD54566975/ssi-sdk/credential/rendering	0.084s
ok  	github.com/TBD54566975/ssi-sdk/credential/schema	1.396s
ok  	github.com/TBD54566975/ssi-sdk/credential/signing	0.073s
ok  	github.com/TBD54566975/ssi-sdk/credential/status	0.186s
?   	github.com/TBD54566975/ssi-sdk/crypto	[no test files]
ok  	github.com/TBD54566975/ssi-sdk/cryptosuite	[13](https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true#step:6:14).113s
ok  	github.com/TBD54566975/ssi-sdk/did	4.389s
ok  	github.com/TBD54566975/ssi-sdk/schema	1.833s
ok  	github.com/TBD54566975/ssi-sdk/util	1.613s
FAIL
Error: running "go test -race -tags=jwx_es[25](https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true#step:6:26)6k ./..."```

**Comment by andorsk** [comment]
Here's a bump from dependabot also failing: https://github.com/TBD54566975/ssi-sdk/runs/7187914608?check_suite_focus=true

**Comment by decentralgabe** [comment]
fixed in #133
